man/man8/zfs.8

```diff
@@ -1329,6 +1334,30 @@ are equivalent to the
 and
 .Sy nodev
 mount options.
+.It Sy dedup Ns = Ns Sy off Ns | Ns Sy on Ns | Ns Sy verify Ns | Ns Sy \
+sha256[,verify] Ns | Ns Sy sha512[,verify] Ns | Ns Sy skein[,verify] Ns | Ns \
+Sy edonr,verify
+Controls whether deduplication in in effect for a dataset. The default value is
+.Sy off .
+The default checksum used for deduplication is
+.Sy sha256
+(subject to change). When dedup is enabled, the dedup checksum algorithm
```

Comment:
"dedup" should be a keyword here.
"dedup" is also repeated twice, should we use FreeBSD zfs(8) wording instead?

> When dedup is enabled, the checksum defined here overrides the checksum property.